jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -591,6 +591,16 @@ public String getContextPathEncoded()
         return _contextPathEncoded;
     }
 
+    /**
+     * Get the context path in a form suitable to be returned from {@link HttpServletRequest#getContextPath()}
+     * @return Returns the encoded contextPath, or empty string for root context
+     */
+    public String getRequestContextPath()
```

Comment:
Interesting question, because I think `ContextHandler.Context.getContextPath()` is actually wrong in that it should be returning the encoded path. 
Eitherway, I think we should only have one of the two methods.... stand by....

Reply:
I think technically it would be`/this%252fpath` and should be ok other than being confusing.
But we should test and I expect some nasties.

Reply:
But I'd prefer not to hold up this PR for that.